app/controllers/items_controller.rb

```diff
@@ -68,7 +68,7 @@ def set_item
     end
 
     # Never trust parameters from the scary internet, only allow the white list through.
-    def item_params
+    def item_params(c = 10, *d, a, b)
```

Comment:
__[method_definition_validator]__
* Lint/DefinitionValidator: Not enough arguments. Did you forget the following arguments? [a, b] at [app/controllers/items_controller.rb#L27](/actcat/sideci_rails_sandbox/blob/8f814ddecda9c71f1321383f85407d15ea95f368/app/controllers/items_controller.rb#L27)
* Lint/DefinitionValidator: Not enough arguments. Did you forget the following arguments? [a, b] at [app/controllers/items_controller.rb#L44](/actcat/sideci_rails_sandbox/blob/8f814ddecda9c71f1321383f85407d15ea95f368/app/controllers/items_controller.rb#L44)